sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/ImportCertificateOptions.java

```diff
@@ -78,17 +78,17 @@ public Boolean isEnabled() {
      * Get the management policy for the certificate.
      * @return the management policy
      */
-    public CertificatePolicy getCertificatePolicy() {
-        return this.certificatePolicy;
+    public CertificatePolicy getPolicy() {
+        return this.policy;
     }
 
     /**
      * Set the management policy for the certificate.
-     * @param certificatePolicy the management policy for the certificate
+     * @param policy the management policy for the certificate
      * @return the updated CertificateImportOptions itself
      */
-    public ImportCertificateOptions setCertificatePolicy(CertificatePolicy certificatePolicy) {
-        this.certificatePolicy = certificatePolicy;
+    public ImportCertificateOptions setPolicy(CertificatePolicy policy) {
```

Comment:
Is this covered from the feedback?

Reply:
yes, [here](onenote:https://microsoft.sharepoint.com/teams/AzureDeveloperExperience/Shared%20Documents/Service%20-%20KeyVault/Azure%20SDK%20-%20Service%20-KeyVault/Release.one#Certificates%20consistency%20meeting%203&section-id=727d79ca-765a-4572-a0e9-bf7323fb398b&page-id=88d10b80-647a-4ca2-aa1c-dd6c53d71735&end)